cloudmarker/stores/emailstore.py

```diff
@@ -0,0 +1,68 @@
+"""email store plugin."""
```

Comment:
Also since this plugin will now be alert plugin can you please make a minor change so that path and class name looks like: `cloudmarker/alerts/emailalert.py` & `EmailAlert`

Reply:
Please also write the module level docstrings with correct capitalization. Please see `MongoDBStore`, `FileStore`, etc. as examples. These docstrings are used to generate project documentation that gets published automatically at https://cloudmarker.readthedocs.io/ , so it is all the more important that these docstrings use correct capitalization in particular and look good in general.

Reply:
> Also since this plugin will now be alert plugin can you please make a minor change so that path and class name looks like: `cloudmarker/alerts/emailalert.py` & `EmailAlert`

I am a little confused here.  As per your comment `cloudmarker/alerts/emailalert.py`, do you want the alerts to be in the different alert package than the store package ?

Reply:
If a module is going to be used an alert plugin, I think it makes sense to have it in `cloudmarker.alerts` package.